parts/k8s/kubernetesconfigs.sh

```diff
@@ -242,7 +242,6 @@ function ensureDocker() {
     DOCKER_JSON_FILE=/etc/docker/daemon.json
     wait_for_file 1200 1 $DOCKER_JSON_FILE || exit $ERR_FILE_WATCH_TIMEOUT
     systemctlEnableAndStart docker
-    retrycmd_if_failure 6 1 10 docker pull busybox # pre-pull busybox, but don't exit if fail
```

Comment:
Should we run this if golden image isn't being used?

Reply:
(in the `if $FULL_INSTALL_REQUIRED; then ` block)

Reply:
It was just a convenience (no error thrown). We'll assume use-cases that don't use the VHD image will have outbound internet access, and so the probe script's usage of `docker run busybox` will simply download the image first.